src/contract.rs

```diff
@@ -17,6 +22,8 @@ pub fn instantiate(
         count: msg.count,
         owner: info.sender.clone(),
     };
+    set_contract_version(deps.storage, CONTRACT_NAME, CONTRACT_VERSION)?;
```

Comment:
This would also be a nice one to add back to the template

Reply:
Done.